PluginDirectories/1/npm.bundle/info.json

```diff
@@ -0,0 +1,6 @@
+{
+	"name": "npmsearch",
```

Comment:
for future reference, the `name` field needs to be the same as the name of the folder (e.g. `npm`)
I'll fix it for now :)